primerpedia.js

```diff
@@ -19,11 +19,17 @@
  */
 
 var wikipediaUrl = "https://en.wikipedia.org";
-var apiUrl = wikipediaUrl + "/w/api.php?";
-var apiExtractsQuery = "action=query&prop=extracts&exintro&indexpageids=true&format=json";
 var requestTimeoutInMs = 3000;
 var requestCallbackName = "requestCallback";
 
+var apiUrl = wikipediaUrl + "/w/api.php?";
+
+var apiExtractsQuery = "action=query&prop=extracts&exintro&indexpageids=true&format=json";
```

Comment:
Perhaps naming this variable `articleIntroQuery` would be clearer and more in line with the names used for the other variables.